mujinvisioncontrollerclient/visioncontrollerclient.py

```diff
@@ -331,7 +340,7 @@ def StopTask(self, taskId=None, taskIds=None, taskType=None, taskTypes=None, cyc
             command['cycleIndex'] = cycleIndex
         return self._ExecuteCommand(command, timeout=timeout, fireandforget=fireandforget)
 
-    def ResumeTask(self, taskId=None, taskIds=None, taskType=None, taskTypes=None, cycleIndex=None, waitForStop=True, fireandforget=False, timeout=2.0):
+    def ResumeTask(self, taskId=None, taskIds=None, taskType=None, taskTypes=None, cycleIndex=None, waitForStop=_deprecated, fireandforget=False, timeout=2.0):
         # type: (Optional[str], Optional[List[str]], Optional[str], Optional[List[str]], Optional[str], Optional[bool], bool, float) -> Optional[Dict[str, List[str]]]
```

Comment:
Shouldn't the last two be also `Optional[X]` ?

Reply:
No. `Optional` type in python means that the parameter can be None. This is different from "optional" in the docstring below where it tells if user has to provide some value or not.

Also, while underlying `_ExecuteCommand` may work when `fireandforget == None` we don't want to encourage users to use None for those parameters without a specific reason. Less Nones, less uncertainty, the better.

Reply:
Oh, now I remember. Thx